contracts/PlanManager.sol

```diff
@@ -141,8 +138,6 @@ contract PlanManager is Initializable, OwnableUpgradeable, IPlanManager {
     function updatePlanTemplateMetadata(uint256 templateId, bytes32 metadata) external onlyOwner {
         if (v2templates[templateId].period > 0) {
```

Comment:
Looks it is alright to have `v2templates` for templates state, but from code the level it would be good to use `struct PlanTemplate` other than `struct PlanTemplateV2`

Reply:
I leave it to avoid code change mistakes. Since I have deployed the contract using this version, I suggest we keep it.